### PR TITLE
Update 2020-07-31-ji20a.md

### PR DESCRIPTION
**Pull request by Jxb814**
Fixed some mistakes in the paper's name, abstract and author info. Had approval from Benjamin Recht.

### CODE REVIEW

**Review by lawrennd — APPROVED** [review]
Thanks for submitting the update Xunbi. Confirm this matches the PDF, although I notice that there are a couple of issues still with the names, missing accents on Tamas's name and also an 's'. I'll fix that in a moment.

File: _posts/2020-07-31-ji20a.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
 month: 0
-tex_title: Feed-forward Neural Network with Trainable Delay and Application to Car-following
+tex_title: Feed-forward Neural Networks with Trainable Delay
 firstpage: 127
 lastpage: 136
 page: 127-136
 order: 127
 cycles: false
-bibtex_author: Ji, Xunbi and Avedisov, Sergei and Molnar, Tama and Orosz, Gabor
+bibtex_author: Ji, Xunbi and Molnar, Tamas and Avedisov, Sergei and Orosz, Gabor
```
Comment (lawrennd) [inline _posts/2020-07-31-ji20a.md:15]:
```suggestion
bibtex_author: Ji, Xunbi and Moln\'ar, Tam\'as and Avedisov, Sergei and Orosz, Gabor
```

File: _posts/2020-07-31-ji20a.md [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
 author:
 - given: Xunbi
   family: Ji
+- given: Tamas
```
Comment (lawrennd) [inline _posts/2020-07-31-ji20a.md:19]:
```suggestion
- given: Tamás
```

File: _posts/2020-07-31-ji20a.md [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
+  family: Molnar
```
Comment (lawrennd) [inline _posts/2020-07-31-ji20a.md:20]:
```suggestion
  family: Molnár
```

**Review by lawrennd — APPROVED** [review]
Add accents to Tamás's name.

File: _posts/2020-07-31-ji20a.md (hunk 1 above)
Comment (Jxb814) [reply _posts/2020-07-31-ji20a.md:15]: Thank you! 
I thought it was the format to drop the accent and middle name...If we add the accent, we need one more accent in G\'abor. Also for the middle names, maybe it is good to have them too.
```suggestion
bibtex_author: Ji, Xunbi A. and Moln\'ar, Tam\'as G. and Avedisov, Sergei S. and Orosz, G\'abor
```

Comment (lawrennd) [reply _posts/2020-07-31-ji20a.md:15]: Thanks for adding Gabor's accent. 

Update with middle initials was perfect also. Thank you!

**Pull request merged**